### ISSUE TITLE
IE (Garden Cloche) Support Not Working?

### ISSUE THREAD

**Issue by ryu777mtg** [Focamacho/MysticalCreations#7]
Minecraft 1.12.2
Forge 14.23.5.2854
Avaritia 1.1x 3.3.0.37
Mystical Adaptations 1.8.5
Mystical Agradditions 1.3.2
Mystical Agriculture 1.7.5
Mystical Agriculture Tiered Crystals 1.0.1-hotfix
Mystical Creations 1.6

Looking in config there is support for it but any seed I've created will not work with the Immersive Engineering Garden Cloche. The items can be inserted to the Cloche but seeds will not grow?

Example of seed config:
infinity;6;D8C7C3;avaritia:block_resource:1;null

This of course uses the Infinity Block from Avaritia as the Crux, I can insert the block and custom seed into Cloche but nothing happens and yes the Cloche has water and Energy.

This setting is also set within the config:

compat {
    # Immersive Engineering Garden Cloche Compat [default: true]
    B:IMMERSIVE_CLOCHE=true
}

Am I missing something? Or is it possible that one of the other Mystcal mods is interfering with this one?

**Comment by focamacho** [comment]
Oh, when I did some code rework a while ago I forgot to change the Garden Cloche compat.
Thanks for the report!
I have uploaded a new version on CurseForge fixing this issue: https://www.curseforge.com/minecraft/mc-mods/mystical-creations/files/3221155

Another known issue for Garden Cloche is that it don't respect the Crop color (because this mod doesn't use a normal texture for crops, they are colored through code), so the crop is always white. I still need to do a workaround for that, just need to know how, maybe generating textures at runtime.
I'll keep this issue open for me to remember to fix the color issue.

Thanks again for the report!

**Comment by ryu777mtg** [comment]
Awesome and thank you so much it's working as intended now.
Though this might be a little off topic is there any chance of getting compatibility for Agricraft Crops Sticks?
This would be a nice feature to have as well.

**Comment by focamacho** [comment]
Yeah but I'll need to fix the color issue first, or it will happen when planting a Mystical Creations crop on an Agricraft Crops Sticks.
